features/git-town/configuration/configuration_reset.feature

```diff
@@ -0,0 +1,36 @@
+Feature: resetting the configuration
+
+  As a user wishing to remove all Git Town settings from a repository
+  I want to be able to cleanly reset the Git Town configuration with a single command
+  So that I can continue working on actual work and stay productive
```

Comment:
This user story makes little sense to me. Here is my take:

```cucumber
As a user having messed up the Git Town configuration beyond repair
I want an easy way to cleanly reset the Git Town configuration completely
So that I can start over and configure it right this time.